concordium-sdk/src/main/java/com/concordium/sdk/transactions/smartcontracts/parameters/ListParam.java

```diff
@@ -0,0 +1,47 @@
+package com.concordium.sdk.transactions.smartcontracts.parameters;
+
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.transactions.InitName;
+import com.concordium.sdk.transactions.ReceiveName;
+import com.concordium.sdk.transactions.smartcontracts.Schema;
+import com.concordium.sdk.transactions.smartcontracts.SchemaParameter;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Getter;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Wrapper for passing lists as smart contract parameters.<p>
```

Comment:
`<p>` ?

Reply:
This should be used only when actual parameter is a list/vector right? Otherwise if the list is somewhere contained within the parameter, then it's fine to just use a plain old `List<Foo>` right?

Reply:
Correct, this is for something like [OperatorOfQueryParams](https://github.com/Concordium/concordium-rust-smart-contracts/blob/bc7e7ae7db029e78c0f1e965c968ca1364f4ce1c/concordium-cis2/src/lib.rs#L1190)
I believe the `#[concordium(transparent]` results in the expected JSON being "just the list" `[OperatorOfQuery, OperatorOfQuery]` and not something like `{queries: [OperatorOfQuery, OperatorOfQuery]}`.  
`List<Foo>` is usable within other parameters without modification e.g. The `List<TokenIdU32>` in `MintParams`